src/jupyter_contrib_nbextensions/nbextensions/toc2/toc2.js

```diff
@@ -165,8 +165,9 @@ var make_link = function(h, num_lbl) {
       callback && callback();
   }
 
-  function setNotebookWidth(cfg, st) {
-    //cfg.widenNotebook  = false; 
+  function setNotebookWidth(cfg, st, sideBarIsToggling) {
+    //cfg.widenNotebook  = true; 
+    var sideBarIsToggling = typeof sideBarIsToggling !== 'undefined' ? sideBarIsToggling : false;
```

Comment:
this `sideBarIsToggling` variable doesn't actually seem to get used anywhere - what is it intended for?